### ISSUE TITLE
Support for commands/queries

### ISSUE THREAD

**Issue by hannuterasloihde** [kaareseras/node-red-contrib-kusto#3]
Currently the node supports only ingestion of data. Would it be possible to add execution of data explorer queries and commands?

**Comment by kaareseras** [comment]
I guess it will be possible to add support for commands relative easily. but Im not sure how I would work with a tabular result-set from a query and push that to other nodes

**Comment by hannuterasloihde** [comment]
True, the parsing of the result set is not trivial. In my use cases I'm mainly fetching rowcounts and it would be sufficient to just pass the raw result from Data Explorer to the next node. I could then parse whatever I need from the result with a function node.
So not an ideal general solution, but it would help me a great deal to get rid of executing shell scripts to do the same thing.
